helix-term/src/commands.rs

```diff
@@ -1309,6 +1309,32 @@ mod cmd {
         editor.set_status(editor.clipboard_provider.name().into());
     }
 
+    fn change_directory(editor: &mut Editor, args: &[&str], _: PromptEvent) {
+        let dir = match args.first() {
+            Some(dir) => dir,
+            None => {
+                editor.set_error("target directory not provided".into());
+                return;
+            }
+        };
+
+        if let Err(e) = ::std::env::set_current_dir(dir) {
```

Comment:
Prefixing with `::` is not really useful here because this is not inside a macro. You also didn't prefix at the other places, so to constant, I suggest to not prefix with `::`.

```suggestion
        if let Err(e) = std::env::set_current_dir(dir) {
```